docs/build/linux-build.md

```diff
@@ -3,22 +3,24 @@
 ```sh
 git clone https://github.com/ostis-apps/nika
 cd nika
+git submodule update --init --recursive
```

Comment:
There is submodule update in **install_submodules** script, I am not sure we need to duplicate it. Seems like we need to fix script. Ask @NikitaZotov or @FallenChromium